test/parallel/test-dgram-send-callback-multi-buffer.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+
+var fs = require('fs');
+var dgram = require('dgram');
+var callbacks = 0;
+var client, timer, buf1, buf2;
+
+buf1 = new Buffer(256);
+buf1.fill('x');
+buf2 = new Buffer(256);
+buf2.fill('y');
+
+client = dgram.createSocket('udp4');
+
+client.on('listening', function() {
+  client.send([buf1, buf2], common.PORT, '127.0.0.1', function(err, bytes) {
```

Comment:
should I amend the other tests as well? All the dgram tests follows the same convention.